properties_description.csv

```diff
@@ -55,28 +55,21 @@ schema:Person,affiliation,Organization,"An organization that this person is affi
 schema:Person,identifier,URL,"URL identifier, ideally an ORCID ID for individuals, a FundRef ID for funders"
 schema:Person,name,Text,"The name of an Organization, or if separate given and family names cannot be resolved for a Person"
 schema:Person,address,PostalAddress or Text,Physical address of the item.
-schema:Review,reviewAspect,Text,This Review is relevant to this part or facet of the object being review
+schema:Review,reviewAspect,Text,This review is relevant to this part or facet of the object being review
 schema:Review,reviewBody,Text,The actual body of the review
-schema:Role,endDate,Date or Datetime,The end date and time of the item, usually a Role. In ISO 8601 date format
-schema:Role,roleName,Text,URL,"A role played, performed, or filled by a person or organization. Suggested values: ""Design"" (algorithm), ""Architecture"", ""Debugging"", ""Maintenance"", ""Coding"", ""Documentation"", ""Testing"", ""Support"", ""Management""."
-schema:Role,startDate,Date or Datetime,The start date and time of the item, usually a Role. In ISO 8601 date format
-codemeta:SoftwareSourceCode,softwareSuggestions,SoftwareSourceCode,"Optional dependencies , e.g. for optional features, code development, etc."
+schema:Role,endDate,Date or Datetime,"The end date and time of the item, usually a Role. In ISO 8601 date format"
+schema:Role,roleName,Text,"A role played, performed, or filled by a person or organization. Suggested values: ""Design"" (algorithm), ""Architecture"", ""Debugging"", ""Maintenance"", ""Coding"", ""Documentation"", ""Testing"", ""Support"", ""Management""."
```

Comment:
```suggestion
schema:Role,roleName,Text or URL,"A role played, performed, or filled by a person or organization. Suggested values: ""Design"" (algorithm), ""Architecture"", ""Debugging"", ""Maintenance"", ""Coding"", ""Documentation"", ""Testing"", ""Support"", ""Management""."
```

Reply:
Thank you. Should I make this one `Text or URL` instead then?